### ISSUE TITLE
针对transfer性能的问题； Regarding the transfer performance issue,

### ISSUE THREAD

**Issue by zisuina** [nvidia-cosmos/cosmos-transfer2.5#234]
我想训练12V multi view的transfer的效果；我训练了12V的720P的predict模型4000step, 8*8 gpu；我的transfer使用这作为backbone, 进行训练，训练大概3500step; 4*8gpu；我目前遇到两个问题： 1. 控制的收敛慢，控制起不到太大作用；请问是这是predict部分和control部分有拉扯导致的吗？我增大lr收敛效果会好一点 2 前面的7V比较快的收敛， 鱼眼的效果一直会有色彩不一致的问题；这是我模型的从始至终遇到的鱼眼数据不够的问题吗？3，我transfer的训练是训练480p启动720p的predict的模型吗？ 请问，针对这两个问题有什么建议呢？

I trained a 12V 720P predict model for about 4000 steps using 8×8 GPUs. Then I used this model as the backbone for transfer training, which has currently been trained for around 3500 steps using 4×8 GPUs.

At the moment, I am encountering three issues:

The control branch converges very slowly, and the control effect is not very significant. Could this be caused by a conflict or competition between the predict part and the control part during training? I noticed that increasing the learning rate seems to improve the convergence somewhat.
In the previous 7V experiments, the model converged relatively quickly, but the fisheye views consistently showed color inconsistency issues (load 12v predict model). Could this be due to insufficient fisheye training data throughout the entire training process?  

For the transfer training, should I train the transfer model at 480p while initializing it from the 720p predict model?
Do you have any suggestions or recommendations regarding these three issues?

**Comment by stihuangyuan** [comment]
> 我想训练12V multi view的transfer的效果；我训练了12V的720P的predict模型4000step, 8_8 gpu；我的transfer使用这作为backbone, 进行训练，训练大概3500step; 4_8gpu；我目前遇到两个问题： 1. 控制的收敛慢，控制起不到太大作用；请问是这是predict部分和control部分有拉扯导致的吗？我增大lr收敛效果会好一点 2 前面的7V比较快的收敛， 鱼眼的效果一直会有色彩不一致的问题；这是我模型的从始至终遇到的鱼眼数据不够的问题吗？3，我transfer的训练是训练480p启动720p的predict的模型吗？ 请问，针对这两个问题有什么建议呢？
> 
> I trained a 12V 720P predict model for about 4000 steps using 8×8 GPUs. Then I used this model as the backbone for transfer training, which has currently been trained for around 3500 steps using 4×8 GPUs.
> 
> At the moment, I am encountering three issues:
> 
> The control branch converges very slowly, and the control effect is not very significant. Could this be caused by a conflict or competition between the predict part and the control part during training? I noticed that increasing the learning rate seems to improve the convergence somewhat. In the previous 7V experiments, the model converged relatively quickly, but the fisheye views consistently showed color inconsistency issues (load 12v predict model). Could this be due to insufficient fisheye training data throughout the entire training process?
> 
> For the transfer training, should I train the transfer model at 480p while initializing it from the 720p predict model? Do you have any suggestions or recommendations regarding these three issues?

老哥我也遇到相同问题，方便加个联系方式交流下吗？
